grudge/op.py

```diff
@@ -455,11 +470,13 @@ def weak_local_div(dcoll: DiscretizationCollection, *args):
 
     :arg dd: a :class:`~grudge.dof_desc.DOFDesc`, or a value convertible to one.
         Defaults to the base volume discretization if not provided.
-    :arg vec: a object array of
-        a :class:`~meshmode.dof_array.DOFArray`\ s,
+    :arg vecs: an object array of
+        :class:`~meshmode.dof_array.DOFArray`\s or
+        :class:`~arraycontext.container.ArrayContainer` objects,
```

Comment:
I don't think `_div_helper` deals with anything other than object arrays.

Reply:
I believe this is all cleared up in https://github.com/inducer/grudge/pull/154/commits/a87734006d247785f3c5baf43bde860c46dbd0c9. `_div_helper` is intentionally only working on object arrays, as the container is expected to have vector (obj array) entries.